wagtail/blocks/stream_block.py

```diff
@@ -433,6 +441,17 @@ def deconstruct(self):
         kwargs = self._constructor_kwargs
         return (path, args, kwargs)
 
+    def deconstruct_with_lookup(self, lookup):
+        path = "wagtail.blocks.StreamBlock"
```

Comment:
I think this would be technically incorrect... if you subclass StructBlock, then its canonical module path should be reported as `bakerydemo.base.blocks.PersonBlock` (or whatever). The fact that `deconstruct` / `deconstruct_with_lookup` chooses to use a basic StructBlock instead of the real class is an implementation detail of those methods.

Reply:
Ah, yep. You're right – thanks!